seedfarmer/models/manifests/_module_manifest.py

```diff
@@ -37,6 +37,18 @@ def __init__(self, **data: Any) -> None:
     def upper_snake_case(self) -> str:
         return self._upper_snake_case
 
+    @model_validator(mode="after")
+    def check_value_or_value_from(self) -> "ModuleParameter":
+        value = self.value
+        value_from = self.value_from
+
+        if value is None and value_from is None:
```

Comment:
I like this!!! 
Should we be raising a seedfarmer-specific error (seedfarmer.errors.InvalidManifestError) or is a ValueError correct?

Reply:
I can change it to `seedfarmer.errors.InvalidManifestError`, but whatever that error is it will end wrapped into `pydantic_core.ValidationError`:

```
self = ModuleManifest()
data = {'name': 'test-module-1', 'parameters': [{'moduleMetadata': {'group': 'test-group', 'key': 'param-key', 'name': 'test-module-1'}, 'name': 'param1'}], 'path': 'modules/test-module', 'targetAccount': 'primary', ...}

    def __init__(self, **data: Any) -> None:
>       super().__init__(**data)
E       pydantic_core._pydantic_core.ValidationError: 1 validation error for ModuleManifest
E       parameters.0
E         Value error, value or value_from must be provided for parameter param1 [type=value_error, input_value={'name': 'param1', 'modul...1', 'key': 'param-key'}}, input_type=dict]
E           For further information visit https://errors.pydantic.dev/2.5/v/value_error
```